### ISSUE TITLE
Board structures

### ISSUE THREAD

**Issue by MichalisKas** [opencooperativeecosystem/kanban#4]
Right now all boards are shown, with no particular sequence or structure.

Regarding the front page, where the member's projects are shown I propose to have a structure depending on the membership (administrator, coordinator, participant etc). The rest of the available projects could be shown on a separate tab.

Regarding the Plans, I propose to have a structure, more or less like the one used in Gitlab:
Several columns categorised according to the status of the plan (to-do, doing, done etc), while inside the columns you may set-up the plan according to their priority (or automatically according to their date).

**Comment by MichalisKas** [comment]
But then I noticed these columns here, which I don't know how are structured...
![screenshot-2017-12-18 react app](https://user-images.githubusercontent.com/20679274/34100399-f9bed73a-e3ea-11e7-8822-39b7183ce57f.png)

**Comment by fosterlynn** [comment]
>But then I noticed these columns here, which I don't know how are structured...

@MichalisKas on that Plan page, if the processes (bins) were connected to each other through resource flows, you would see them as you see in that screen shot.  Then they would be the form of kanban where the bins are ordered in a meaningful way, and one bin would be dependent on completing the bin before it.  

(That would be according to the internal value flow structure of the processes.  So each Plan could have a different set of columns, depending on the recipe or just the way the processes are hooked together, instead of all being the same as in to-do, doing, etc.  You would use that when you have dependencies between the processes, especially when done by different people - simple example: creating a post, getting approval for the post, translating the post, posting the post.  Or thinking about circular economy, you could have much more complex sets of dependencies, where products are produced, maybe from raw materials produced in other processes, and then transported to a node, then distributed.... that is the kind of use case where this value flows structure will be really helpful, we think.)

In your example though, those processes are not dependent on each other, so maybe would better be shown vertically?  Or not?  Or shown in separate plans?  In any case, I think it would be good to visually see the difference between processes that are in an actual resource dependency flow, and processes that are just conveniently grouped in the same plan because say they are what you do as a facilitator.  I started an issue to discuss that #2.

Hope this was at least sort of clear. :blush:

**Comment by ivanminutillo** [comment]
> Regarding the front page, where the member's projects are shown I propose to have a structure depending on the membership (administrator, coordinator, participant etc). The rest of the available projects could be shown on a separate tab.

Ok, since the homepage is currently divided in _**your plans**_ (the list of all plans the agent is involved with) and _**your organizations**_ , we can show first the **_your plans_** section, in a way that the agent can see in a matter of second the works s/he is currently involved with (and adding some secondary infos: status,  number of processes, number of members etc)
and then as you said we can divide the **_your organization_** section depending on the membership.

**Comment by MichalisKas** [comment]
Agree. But even in Your Plans section there may be too many so we got to arrange them somehow.
By due date?
By membership status?

**Comment by fosterlynn** [comment]
Not recommending, just commenting.  One possible way to arrange the Your Plans section is like @bhaugen 's fake-kanban in the OCP work app.  We know something about the stages of the plans based on data.  Then that section would be a broader scope kanban type thing.  Perhaps with less data in the bins than this example?

![bob-kanban](https://user-images.githubusercontent.com/3776081/34207243-4e359b18-e54f-11e7-85fd-61b3fc0447ac.png)

**Comment by ivanminutillo** [comment]
I just implemented this visualization as the standard one on each process in the 0.0.5 version of the kanban app.
I left the choice to switch to a resource flow version on the process title
